src/image.rs

```diff
@@ -23,6 +23,13 @@ pub struct YamlSpec {
     key: String,
 }
 
+#[derive(Debug, PartialEq, Serialize, Deserialize)]
+pub struct ExecSpec {
+    command: String,
+    args: Vec<String>,
+    tag: String,
```

Comment:
tag is being used here, where name+tag is more what is meant.

Perhaps we should have `name` and `tag` (I was thinking of adding `tag` to the docker build variant), or this should just be `name` or `image` to reflect that its more than just the tag.

Reply:
Happy to go with `image`